glide.yaml

```diff
@@ -1,4 +1,4 @@
-package: github.com/projectcalico/calico/calico_upgrade
+package: github.com/projectcalico/calico-upgrade
```

Comment:
Need to do a glide up as well?

Reply:
the glide.lock should be nice and fresh no?